src/test/java/org/myrobotlab/integration/InMoovScriptTest.java

```diff
@@ -2,116 +2,113 @@
 
 import static org.junit.Assert.assertNotNull;
 
-import java.io.File;
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
 
+import org.junit.BeforeClass;
+import org.junit.Ignore;
 import org.junit.Test;
 import org.myrobotlab.io.FileIO;
+import org.myrobotlab.net.Http;
 import org.myrobotlab.service.InMoov;
 import org.myrobotlab.service.Python;
 import org.myrobotlab.service.Runtime;
 import org.myrobotlab.test.AbstractTest;
 
+
 public class InMoovScriptTest extends AbstractTest {
 
-  private String scriptRoot = "src/test/resources/InMoov";
+  boolean testLocal = true;
+
+  // set this to null to check the checked in scripts
+  static String scriptRoot =  null;// "C:\\mrl\\mrl.develop\\pyrobotlab\\service\\";
+  
+  static String branch = "develop";
+  
+  static boolean virtual = true;
+
+  static List<Script> scripts = new ArrayList<>();
+
+  @BeforeClass
+  public static void setUpBeforeClass() throws Exception {
+
+    addScript("InMoov.py", scriptRoot);
+    addScript("InMoovArm.py", scriptRoot);
+    addScript("InMoovEyelids.py", scriptRoot);
+    addScript("InMoovFingerStarter.py", scriptRoot);
+    addScript("InMoovGestureCreator.py", scriptRoot);
+    addScript("InMoovHand.py", scriptRoot);
+    addScript("InMoovHead.py", scriptRoot);
+    addScript("InMoov.py", scriptRoot);
+    addScript("InMoovTorso.py", scriptRoot);
 
-  // Test the inmoov minimal script.
-  @Test
-  public void testInMoovMinimal() throws IOException {
-    if (printMethods)
-      System.out.println(String.format("Running %s.%s", getSimpleName(), getName()));
-    // TODO: move these scripts to test resources
-    String inmoovScript = scriptRoot + "/InMoov.minimal.py";
-    File f = new File(inmoovScript);
-    System.out.println("IN MOOV SCRIPT: " + f.getAbsolutePath());
-    // InputStream is = this.getClass().getResourceAsStream(inmoovScript);
-    String script = FileIO.toString(inmoovScript);
-    // String script = new String(FileIO.toByteArray(is));
-    Python python = (Python) Runtime.createAndStart("python", "Python");
-    // python.execAndWait(script);
-    python.exec(script);
-    InMoov i01 = (InMoov) Runtime.getService("i01");
-    // Assert something
-    assertNotNull(i01);
   }
 
-  // Test the inmoov minimal arm script.
-  @Test
-  public void testInMoovMinimalArm() throws IOException {
-    if (printMethods)
-      System.out.println(String.format("Running %s.%s", getSimpleName(), getName()));
-    if (!isHeadless()) {
-      Runtime.start("gui", "SwingGui");
+  public static void addScript(String filename, String root) throws IOException {
```

Comment:
I don't like relying on external resources in unit tests.  What is the intention of this?  to test what the current latest on develop is?  that means the mrl build will fail if there's a new error introduced into the scripts in the pyrobotlab repo... I don't think that's right.

Reply:
All service scripts have moved to the myrobotlab repo NO LONGER the pyrobotlab
(its a good thing ...)
all that crap of trying to keep on the same branch and get resources across repos is done... however the Test have not all been updated..
Its lots of work and I'm not done with it ...